ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/services/PartyManagementServiceAuthorization.scala

```diff
@@ -66,7 +66,7 @@ private[daml] final class PartyManagementServiceAuthorization(
 
   override def close(): Unit = service.close()
 
-  override def updateUserIdentityProviderId(
+  override def updatePartyIdentityProviderId(
```

Comment:
@mziolekda @pbatko-da  : looks benight, but the question is whether has this service already been released? and if yes, with what guarantees?

i.e., are we sure that this change won't break our customer's code?